test/system/post_test.rb

```diff
@@ -31,6 +31,25 @@ class PostTest < ApplicationSystemTestCase
 
   end
 
+  test 'adding tags to the post' do
+    visit '/wiki/wiki-page-path/comments'
+
+    find('a[data-target="#loginModal"]', text: 'Login').click()
+
+    fill_in 'user_session[username]', with: 'jeff'
+    fill_in 'user_session[password]', with: 'secretive'
+
+    click_on 'Log in'
+
+    find('a#tags-open').click()
+
+    find('.tag-input').set('nature').native.send_keys(:return)
```

Comment:
This looks good! Could you explain what the `.native.send_keys(:return)` does? I am just curious!

Reply:
As you can probably guess it emits the "Enter" keyboard key. In order to add a tag, the user has to press the "Enter" key since this is a test, I have to emit it somehow.

Reply:
Just to understand, why wouldn't the `fill_in` method work here?

Reply:
As the name suggests `fill_in` will just fill the input with some text, but that's it!
It won't trigger any submit/enter events. For a tag to be added, user has to press the "Enter" key.
That's why we need `.native.send_keys(:return)`.

I hope that it's clear now. Thank you for the interest though.

Reply:
Oh alright I get it. We can only use that for forms with submit buttons. I'm not experienced with system testing. Thanks for your help :+1:

Reply:
Anything else? :)